src/profile-logic/old-cleopatra-profile-format.js

```diff
@@ -183,6 +186,29 @@ function _convertThread(
   for (let i = 0; i < thread.markers.length; i++) {
     const marker = thread.markers[i];
     const markerIndex = markers.length++;
+    const data = marker.data;
+    if (data && 'stack' in data) {
```

Comment:
so we used to have the stack in cleopatra already ?

Reply:
Yes we did! Only for styles markers, though. I'm not sure when the reflow cause was added. The old cleopatra exposed the cause callstack in two ways: With a regular tooltip on the markers in the header, and in the markers list where the "syncProfile" call tree was displayed as a subtree of the marker node.